crates/bevy_dev_tools/src/picking_debug.rs

```diff
@@ -0,0 +1,323 @@
+//! Text and on-screen debugging tools
+
+use core::cmp::Ordering;
+use core::fmt::{Debug, Display, Formatter, Result};
+
+use bevy_app::prelude::*;
+use bevy_asset::prelude::*;
+use bevy_color::prelude::*;
+use bevy_ecs::prelude::*;
+use bevy_picking::backend::HitData;
+use bevy_picking::hover::HoverMap;
+use bevy_picking::pointer::{Location, PointerId, PointerPress};
+use bevy_picking::prelude::*;
+use bevy_picking::{pointer, PickSet};
+use bevy_reflect::prelude::*;
+use bevy_render::prelude::*;
+use bevy_text::prelude::*;
+use bevy_ui::prelude::*;
+use bevy_utils::tracing::{debug, trace};
```

Comment:
Simply updating the branch doesn't fix the issue here - you have to change this use statement as well.
```suggestion
use tracing::{debug, trace};
```

Reply:
I'm aware. I'm working on the rest of the changes before pushing again

Reply:
My bad, then.

Reply:
All good, no worries 🙂